src/policy/copyspace.rs

```diff
@@ -100,6 +100,15 @@ impl<VM: VMBinding> SFT for CopySpace<VM> {
         let worker = worker.into_mut::<VM>();
         self.trace_object(queue, object, self.common.copy, worker)
     }
+
+    fn debug_get_object_info(&self, object: ObjectReference) -> String {
+        format!(
+            "{}: {:?}, {:?}",
+            self.name(),
+            object_forwarding::debug_get_object_forwarding_info::<VM>(object),
+            crate::policy::sft::debug_get_object_global_info(object)
+        )
```

Comment:
The same is here.  We can put three `println!` statements here and print three pieces of information separately.